README.MD

```diff
@@ -23,28 +24,28 @@ Please note these demos should be considered as CSS "Proofs of Concepts". They m
 ## Quick links
 
 1. [Accordion / Toggle](#Accordion)
-1. [Carousel](#Carousel)
-1. [Counter of checked check-boxes](#Counter)
-1. [Flip on click](#Flip)
-1. [Floating label on Textfield](#Floating)
-1. [Font-Face (Latin)](#Font)
-1. [Info on hover/ Popover](#Info)
-1. [Image Gallery](#Image)
-1. [Menu](#Menu)
-1. [Mobile menu off canvas](#Mobile)
-1. [Burger menu](#BurgerMenu)
-1. [Fancy menu](#FancyMenu)
-1. [Modal/Popup](#Modal)
-1. [Mouse tracking](#Mouse)
-1. [Parallax scrolling](#Parallax)
-1. [Tabs](#Tabs)
-1. [Todo List](#Todo)
-1. [Tooltips](#Tooltips)
-1. [Treeview](#Treeview)
-1. [Twitter Heart Animation](#Twitter)
-1. [Dynamic Image Colorizing](#ImageColorizing)
-1. [Ripple Effect](#RippleEffect)
-1. [Responsive Counter Showing # of Items That Didn't Fit Screen](#ResponsiveItemShowcase)
+2. [Carousel](#Carousel)
```

Comment:
`1.` requires no maintaince. GH numerates list items itself. It also has `#.` notation for them, so no misinformation like `1.` in the plain text form.